installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -22,9 +23,9 @@ static bool g_Silent = false;
 
 namespace // Strings in this namespace should not be localized
 {
-    const wchar_t APPLICATION_ID[] = L"PowerToysInstaller";
-    const char EXE_LOG_FILENAME[] = "powertoys-bootstrapper-exe-" STR(VERSION_MAJOR) "." STR(VERSION_MINOR) "." STR(VERSION_REVISION) ".log";
-    const char MSI_LOG_FILENAME[] = "powertoys-bootstrapper-msi-" STR(VERSION_MAJOR) "." STR(VERSION_MINOR) "." STR(VERSION_REVISION) ".log";
+  const wchar_t APPLICATION_ID[] = L"PowerToysInstaller";
```

Comment:
Please use 4 spaces indentation.

Reply:
I've just noticed that `.clang-format` is located in `src/`, while this code is located under `installer/PowerToysBootstrapper/bootstrapper/`. That's the reason why VS keeps using my local style instead of the one specified in `src/.clang-format`.